src/network/web3bridge.js

```diff
@@ -62,18 +61,18 @@ class Web3Bridge {
         });
     }
 
-    async start() {}
+    async start() { }
 
     async loadPointContract(contractName, at) {
+
         if (!(contractName in abisByContractName)) {
+
             const buildDirPath = path.resolve(
-                resolveHome(config.get('datadir')),
```

Comment:
The artefacts should be stored in the `datadir` for now. Later the whole workflow here will be changed.